diesel/src/query_builder/select_statement/dsl_impls.rs

```diff
@@ -292,12 +307,13 @@ where
             self.limit_offset,
             group_by,
             self.locking,
+            self.having,
         )
     }
 }
 
 impl<F, S, W, O, LOf, Lock> LockingDsl<Lock>
-    for SelectStatement<F, S, NoDistinctClause, W, O, LOf>
+    for SelectStatement<F, S, NoDistinctClause, W, O, LOf, NoGroupByClause, NoLockingClause>
```

Comment:
That change shouldn't be necessary as both `GB` and `LC` correspond to their `No*Clause` variant, right?